hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/RootEndpoint.java

```diff
@@ -61,6 +62,9 @@ public Response get()
         throw e;
       }
 
+      OzoneVolume volume = getVolume();
+      response.setOwner(new S3Owner(volume.getName(), volume.getName()));
```

Comment:
this line doesn't look right. The owner is the user owner of the volume, not the name of the volume